mne/beamformer/_lcmv.py

```diff
@@ -441,7 +442,7 @@ def _lcmv_source_power(info, forward, noise_cov, data_cov, reg=0.05,
     # Tikhonov regularization using reg parameter to control for
     # trade-off between spatial resolution and noise sensitivity
     # This modifies Cm inplace, regularizing it
-    Cm_inv, d, _ = reg_pinv(Cm, reg)
+    Cm_inv, d, _ = _reg_pinv(Cm, reg, rank=rank)
```

Comment:
@wmvanvliet I think you just missed this one as every other `Cm_inv, ..., _reg_pinv` preceded by a Tikhonov comment has `rank=rank`, but maybe I'm missing a subtlety here

Reply:
Ah yes, I missed this one.